src/test/integration/codeAction.test.ts

```diff
@@ -29,6 +29,9 @@ suite('code actions', () => {
       new vscode.CodeAction('Format Document', vscode.CodeActionKind.Source.append('formatAll').append('terraform')),
     ];
 
+    // wait till the LS is ready to acccept a code action request
+    await sleep(1000);
```

Comment:
```suggestion
    await new Promise(r => setTimeout(r, 1000));
```
What do you think about inlining the sleep? Since it is not used anywhere else